jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -0,0 +1,83 @@
+---
+version:
+- Cloud
+---
+= Using OpenID Connect Tokens in Jobs
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to use OpenID Connect ID tokens for access to compatible cloud services.
+:icons: font
+:toc: macro
+:toc-title:
+
+In jobs using a context, CircleCI provides an OpenID Connect ID token in an environment variable. A job can use this to access compatible cloud services without a long-lived credential stored in CircleCI.
+
+toc::[]
+
+== OpenID Connect ID token availability
+
+In CircleCI jobs that use at least one context, the OpenID Connect ID token is available in the environment variable `CIRCLE_OIDC_TOKEN`.
+
+Add a <<contexts/#creating-and-using-a-context,context to a job>> by adding the `context` key to the <<configuration-reference/#workflows,workflows>> section of your `circleci/config.yml` file:
+
+```yaml
+workflows:
+  my-workflow:
+    jobs:
+      - run-tests:
+          context:
+            - my-context
+```
+
+== Setting up your cloud service
+
+_Consult your cloud service's documentation for how to add an identity provider._
```

Comment:
A change from the [old PR](https://github.com/circleci/circleci-docs/pull/6578) that didn't make it over here:

> Consult your cloud service's documentation for how to add an identity provider, for example, AWS's [Creating OpenID Connect (OIDC) identity providers](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) or Google Cloud Platform's [Configuring workload identity federation](https://cloud.google.com/iam/docs/configuring-workload-identity-federation#oidc).

Could we get that in here?